### PR TITLE
Gives rest of command the new IDs

### PR DESCRIPTION
**Pull request by WilliamMurdoch**
I forgot to give the rest of command (HoS, RD, OM, and CMO) the new command IDs. This fixes that... woops.

### CODE REVIEW

**Comment by WilliamMurdoch** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: html/changelogs/CatsinHD - Fix Command IDs.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Gives OM, CMO, HoS, and RD the new command IDs.
```
Comment (FlamingLily) [inline html/changelogs/CatsinHD - Fix Command IDs.yml]: You did actually also give the CE the ID according to the code

File: html/changelogs/CatsinHD - Fix Command IDs.yml [hunk 2]
```diff
@@ -55,4 +55,4 @@ delete-after: True
 # Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
 # Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
 changes:
-  - rscadd: "Gives OM, CMO, HoS, and RD the new command IDs.
+  - rscadd: "Gives OM, CMO, HoS, and RD the new command IDs."
```
Comment (FlamingLily) [inline html/changelogs/CatsinHD - Fix Command IDs.yml:58]: You also gave the new ID to the CE.

Comment (WilliamMurdoch) [reply html/changelogs/CatsinHD - Fix Command IDs.yml:58]: Oh yeah, forgor to add him in there

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**